content/cp_benefits/1_intro.md

```diff
@@ -0,0 +1,17 @@
+---
+image: cp_benefits/modelling.png
+headline: Convection-permitting modeling in Fenno-Scandinavia
+---
+## Benefits and added value of convection-permitting climate modeling over Fenno-Scandinavia
+
+Snow, ice, the long coastline, and the Scandes mountains, they all make the Fenno-Scandinavia a complex region. The box plots show observed distribution of daily mean T2m (top panel) and Pr (bottom) for DJF (left half) and JJA (right half) seasons. 
```

Comment:
Maybe write out the abbreviations (T2m, Pr, DJF, JJA, ...)

Reply:
Also in other panels

Reply:
Thanks for the comment! I will spell out all these abbreviations.